src/components/tag-skill/tag-skill.md

```diff
@@ -0,0 +1,15 @@
+`TagSkill` is the variant of `Tag` for Mavenlink skills.
```

Comment:
We should probably mention that it is similarly intended for use in `TagList`.

Reply:
I can link back to Tag. Lemme figure that out.